tests/models/xlm_roberta_xl/test_modeling_xlm_roberta_xl.py

```diff
@@ -515,6 +516,80 @@ def test_create_position_ids_from_inputs_embeds(self):
         self.assertEqual(position_ids.shape, expected_positions.shape)
         self.assertTrue(torch.all(torch.eq(position_ids, expected_positions)))
 
+    # This test was copied from the common test_eager_matches_sdpa_generate(), but without low_cpu_mem_usage=True.
+    # TODO: Remove this and use the parent method (in common tests) once XLM RoBERTa XL supports low_cpu_mem_usage=True.
```

Comment:
Instead of copy-pasting the whole thing here, we can use `# Copied from` and `#Ignore copy` above the line where `low_cpu_mem_usage=True` is set. This way any updates to the common test will still be propogated here